### PR TITLE
[DEV-57] Documentation for Algorithmia Insights for Java SDK

### PR DESCRIPTION
**Pull request by dinoboy197**
* Included Java API contract
* Linked Java, R, and Python callouts in specific client library links

### CODE REVIEW

File: _pages/clients/java.md [hunk 1]
```diff
@@ -261,6 +261,21 @@ This guide used the the first chapter of [Jack London's Burning Daylight](https:
 
 If you are interested in learning more about working with unstructured text data check out our guide [Introduction to Natural Language Processing](https://algorithmia.com/blog/introduction-natural-language-processing-nlp).
 
+{% if site.enterprise %}
+## Publishing Algorithmia Insights
+
+Inference-related metrics (a feature of [Algorithmia Insights](../../algorithmia-enterprise/algorithmia-insights)) can be reported via using the `reportInsights` method of the Algorithmia client.
```
Comment (bnlb) [inline _pages/clients/java.md]:
```suggestion
Inference-related metrics (a feature of [Algorithmia Insights](../algorithmia-enterprise/insights)) can be reported via using the `reportInsights` method of the Algorithmia client.
```

Comment (dinoboy197) [reply _pages/clients/java.md]: It turns out [product has explicitly asked us to use "Algorithmia Insights" everywhere for branding purposes](https://github.com/algorithmiaio/dev-center/pull/340#pullrequestreview-482276022)!

Comment (bnlb) [reply _pages/clients/java.md]: Ah gotcha, in that case `_pages/algorithmia-enterprise/insights.md` will have to be renamed to `_pages/algorithmia-enterprise/algorithmia-insights.md` so it doesn't 404.

**Review by kennydaniel — APPROVED** [review]
Wording and sample code looks good to me

File: _pages/clients/java.md (hunk 1 above)
Comment (bnlb) [reply _pages/clients/java.md]: `../../algorithmia-enterprise/algorithmia-insights` should be `../algorithmia-enterprise/algorithmia-insights`. The double `../` removes `developers` from the path.

**Review by bnlb — CHANGES_REQUESTED** [review]
Only blocker is that extra `../` on the Java page. Everything else looks great.

File: _pages/clients/java.md (hunk 1 above)
Comment (dinoboy197) [reply _pages/clients/java.md]: Thanks Nathan! Changes made!

Comment (dinoboy197) [reply _pages/clients/java.md]: (all tested and working on https://insights1.enthalpy.click/developers/algorithmia-enterprise/algorithmia-insights)

**Review by bnlb — APPROVED** [review]
🌟

**Pull request merged**